### ISSUE TITLE
Bug: A11y: Link vs text contrast does not meet WCAG guidelines

### ISSUE THREAD

**Issue by mobot11** [SAP/fundamental#1297]
**Describe the bug**

* WCAG requires "A 3:1 contrast between the link text color and the surrounding non-link text color." when color is the only indicator used for links. https://webaim.org/articles/contrast/#only. Can check the allowable contrast levels here: https://webaim.org/resources/linkcontrastchecker/

> Is this issue related to a specific component?

* Any component using default text and links.

> What version of the Fiori Fundamentals are you using?

* ^1.4.3

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

* React-Fundamentals

> What front-end framework are you implementing in, e.g., Angular, React, etc.?

* React

**To Reproduce**
1. Go to https://webaim.org/resources/linkcontrastchecker/
2. Type in the default text color, #32363a and the default link color, #085caf
3. See failure

**Expected behavior**
The default text and link color should have at least a 3:1 ratio for WCAG A guidelines.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: IOS 10.14
 - Browser Chrome
 - Version 72.x

**Additional context**
Possible fixes are adding an underline or changing the hex values so they meet contrast specifications.

Relates to: https://github.com/SAP/fundamental-react/issues/460

**Comment by jbadan** [comment]
Medium buttons (orange) do not meet 4.5:1 color contrast standards as well. 

Medium: insufficient color contrast of 3.03 (foreground color: #e9730c, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1
